### ISSUE TITLE
discord streaming

### ISSUE THREAD

**Issue by regiscy** [Totsukawaii/UndownUnlock#33]
Whenever I inject and right at the moment when the window pops up and ends process of 3rd party software such as discord, it does not close it. I also tested it streaming to my friends and it works. What else should I do to make it undetectable? is it gonna screenshot the stream or detect it? I was thinking since I am streaming to my friends they would just send me the answers on my phone and take a glimpse on my phone and look at the answer.

**Comment by Secrets123212** [comment]
Wait are you able to stream from LockDown Browser without encountering a black screen?

**Comment by regiscy** [comment]
I will do a precheck first

**Comment by regiscy** [comment]
Yes I can stream without a black screen after dll injection, soon ill do the precheck exam test and let you knoiw

**Comment by alany08** [comment]
I can stream even without UndownUnlock LMAO. I three finger swiped (it does detection but doesn't work for me for some reason) to the browser and just screenshared to google meet without a black screen.
